### PR TITLE
PLAT-101531: Update VirtualList and Scroller styles in default samplers

### PR DESCRIPTION
**Pull request by ybsung**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [ ] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added / Resolution
[//]: # (Describe the issue resolved or feature added by this pull request)

According to the theme libraries, the styles of the VirtualList and the Scroller updates.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

If we don't need to align with the theme sampler's UX/GUI, this PR could be discarded.

### Links
[//]: # (Related issues, references)

PLAT-101531

### CODE REVIEW

File: packages/sampler/stories/default/VirtualList.js [hunk 1]
```diff
@@ -54,8 +46,8 @@ storiesOf('UI', module)
 				<VirtualList
 					dataSize={updateDataSize(number('dataSize', VirtualListConfig, defaultDataSize))}
 					horizontalScrollbar={select('horizontalScrollbar', prop.scrollbarOption, VirtualListConfig)}
-					itemRenderer={uiRenderItem(ri.scale(number('itemSize', VirtualListConfig, 72)))}
-					itemSize={ri.scale(number('itemSize', VirtualListConfig, 72))}
+					itemRenderer={uiRenderItem(ri.scale(number('itemSize', VirtualListConfig, 156)))}
+					itemSize={ri.scale(number('itemSize', VirtualListConfig, 156))}
```
Comment (jeonghee27) [inline packages/sampler/stories/default/VirtualList.js]: image

I don't think we should update it. 
Let's update sandstone only.

Comment (MikyungKim) [reply packages/sampler/stories/default/VirtualList.js]: I agree.

**Comment by MikyungKim** [comment]
Could you also take a look at VirtualGridList sample? I think item needs `width=100%`.

File: packages/sampler/stories/default/VirtualList.js (hunk 1 above)
Comment (MikyungKim) [reply packages/sampler/stories/default/VirtualList.js]: Let's not change 72 to 156 here.

File: packages/sampler/stories/default/VirtualList.js [hunk 2]
```diff
@@ -8,21 +8,13 @@ import {storiesOf} from '@storybook/react';
...
 	uiRenderItem = (size) => ({index, ...rest}) => {
-		const itemStyle = {
-			borderBottom: ri.unit(3, 'rem') + ' solid #202328',
-			boxSizing: 'border-box',
-			height: size + 'px'
-		};
-
 		return (
-			<UiItem {...rest} style={itemStyle}>
+			<UiItem {...rest} style={{height: size + 'px'}}>
```
Comment (MikyungKim) [inline packages/sampler/stories/default/VirtualList.js]: let's keep the border here too.

File: packages/sampler/stories/default/VirtualGridList.js [hunk 3]
```diff
@@ -67,7 +67,7 @@ storiesOf('UI', module)
 				horizontalScrollbar={select('horizontalScrollbar', prop.scrollbarOption, VirtualGridListConfig)}
 				itemRenderer={uiRenderItem}
 				itemSize={{
-					minWidth: ri.scale(number('minWidth', VirtualGridListConfig, 180)),
+					minWidth: ri.scale(number('minWidth', VirtualGridListConfig, 100)),
```
Comment (MikyungKim) [inline packages/sampler/stories/default/VirtualGridList.js]: Sorry.. but.. I didn't mean this. Please add `width=100%` to the style.

**Pull request merged**